worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitWorld.java

```diff
@@ -38,6 +38,7 @@
 import com.sk89q.worldedit.util.SideEffectSet;
 import com.sk89q.worldedit.util.TreeGenerator;
 import com.sk89q.worldedit.world.AbstractWorld;
+import com.sk89q.worldedit.world.WorldUnloadedException;
```

Comment:
This is correct - I moved an internal exception that was used in every platform to core